pipe_tools/beam/requirements.py

```diff
@@ -0,0 +1,36 @@
+import logging
+import sys
+
+# From https://beam.apache.org/documentation/sdks/python-dependencies/
+beam_version = "2.11.0"
```

Comment:
@bitsofbits I would add a bit more of documentation so it is easier in the future to understand how this works, also in case they change the format of the page
```
# Class created to parse the dependencies extracted from
# https://beam.apache.org/documentation/sdks/python-dependencies/
# 1st Go to the page and select the version of bean you want to use
# 2nd Copy the table and paste it in the `beam_requirement_text` below
```

Reply:
@enriquetuya : Good idea, I added text to this effect to the module docstring and updated the changelong.